Doc/library/typing.rst

```diff
@@ -2905,6 +2905,38 @@ Introspection helpers
 
    .. versionadded:: 3.8
 
+.. function:: get_protocol_members(tp)
+
+   Return the set of members defined in a :class:`Protocol`.
+
+   ::
+
+      >>> from typing import Protocol, get_protocol_members
+      >>> class P(Protocol):
+      ...     def a(self) -> str: ...
+      ...     b: int
+      >>> get_protocol_members(P)
+      frozenset({'a', 'b'})
+
+   Raise a :exc:`TypeError` for arguments that are not Protocols.
+
+   .. versionadded:: 3.13
+
+.. function:: is_protocol(tp)
+
+   Check if a type is a :class:`Protocol`.
```

Comment:
```suggestion
   Determine if a type is a :class:`Protocol`.
```

Reply:
That's probably better, but the current wording aligns with the one for `is_typeddict`.